### PR TITLE
IT-5040 | OverlayHeader

### PR DESCRIPTION
**Pull request by kvas-damian**

### CODE REVIEW

File: lib/js/components/OverlayHeader/OverlayHeader.stories.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	title: 'Zadanie #123',
+	shortTitle: '#123',
+	eyebrowText: 'Eyebrow',
+	accessorySlot: 'accessory',
+	actionsSlot: 'actions slot',
+	dropdownSlot: 'dropdown slot',
+	eyebrowAccessorySlot: 'eyebrowAccessory',
+	titleLeadingSlot: 'leading',
+	titleTrailingSlot: 'trailing',
+	titleIsLink: false,
```
Comment (rogatty) [inline lib/js/components/OverlayHeader/OverlayHeader.stories.ts:52]:
```suggestion
	isTitleInteractive: false,
```

File: lib/js/components/OverlayHeader/OverlayHeader.vue [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+				<slot name="actions" />
+			</div>
+			<ds-divider
+				class="ds-overlayHeader__divider -mobileHidden"
+				:prominence="DIVIDER_PROMINENCES.STRONG"
+				is-vertical
+			/>
+		</template>
+		<template v-if="$scopedSlots.dropdown">
+			<ds-dropdown boundaries-selector="body">
```
Comment (rogatty) [inline lib/js/components/OverlayHeader/OverlayHeader.vue:59]: I won't be surprised if we'll have to add a prop to make it configurable, but we can add it when needed.

File: lib/js/components/OverlayHeader/OverlayHeader.vue [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+		margin-left: $space-xxs;
+		overflow-x: auto;
+
+		&::after {
+			align-self: stretch;
+			background: linear-gradient(
+				90deg,
+				rgba(var(--raw-gray-50-rgb), 0) 0%,
+				$color-neutral-background 84.26%
+			);
```
Comment (rogatty) [inline lib/js/components/OverlayHeader/OverlayHeader.vue:205]: It's copy pasted from https://github.com/bethinkpl/wnl-platform/blob/f7b949aee683ca593df6b6159409f3bd6f4cfcf0/resources/assets/js/modules/assessments/components/headers/AssessmentHeader.vue#L357-L361 so maybe it's time to extract it as a DS variable or a mixin?

**Review by rogatty — APPROVED** [review]

File: lib/js/components/OverlayHeader/OverlayHeader.vue [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+		/>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/media-queries';
+@import '../../../styles/settings/colors/tokens';
+@import '../../../styles/settings/typography/tokens';
+@import '../../../styles/mixins/flex_overflow_mask';
```
Comment (rogatty) [inline lib/js/components/OverlayHeader/OverlayHeader.vue:91]: Why not kebab-case like other files?

File: lib/js/components/OverlayHeader/OverlayHeader.vue [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+
+$scrollable-mask-width: 20px;
```
Comment (rogatty) [inline lib/js/components/OverlayHeader/OverlayHeader.vue:93]: It's unused.

File: lib/js/components/OverlayHeader/OverlayHeader.vue (hunk 4 above)
Comment (kvas-damian) [reply lib/js/components/OverlayHeader/OverlayHeader.vue]: [`3da9bb8` (#260)](https://github.com/bethinkpl/design-system/pull/260/commits/3da9bb8acdaded82cfa0c106341da0ef46a18ce5)

**Review by rogatty — APPROVED** [review]

**Pull request merged**